### PR TITLE
Disabling buttons of answers once one of them was selected

### LINKED ISSUE #10: Disable other answers once one is selected

### PR DESCRIPTION
**Pull request by Yasserhub**
Closes #10

### CODE REVIEW

File: src/views/answerView.js [hunk 1]
```diff
@@ -4,8 +4,16 @@
  */
 export const createAnswerElement = (key, answerText) => {
   const element = document.createElement('li');
+
   element.innerHTML = String.raw`
-    ${key}: ${answerText};
-  `;
+    <button class="answers" id=${key}>${key}: ${answerText}</button>
```
Comment (arkaitzgarro) [inline src/views/answerView.js:9]: The `id` attribute is not used, can be removed.

**Review by arkaitzgarro — COMMENTED** [review]
Disabling the buttons works now, but if you select the right answer, it shows both the green text and the red box. Can you take a look why is this?

That is right. I think this option is made by Svitlana. This means that the selected answers should be coloured red whether is correct or not. I will try to fix it.

**Comment by arkaitzgarro** [comment]
> The problem is that condition goes every time to "else" because the value of selectedElement and selectedElement are empty.

@Yasserhub  What do you mean they are empty? I see the two elements printed in the console.

Screenshot 2022-10-12 at 16 45 06

**Comment by Yasserhub** [comment]
> Disabling the buttons works now, but if you select the right answer, it shows both the green text and the red box. Can you take a look why is this?
> 
> That is right. I think this option is made by Svitlana. This means that the selected answers should be coloured red whether is correct or not. I will try to fix it.
> 
> The problem is that condition goes every time to "else" because the value of selectedElement and selectedElement are empty.
> 
> export const selectAnswerVariant = (selectedElement, selectedElement) => { console.log("Hello", selectedElement, rightAnswer); if (selectedElement === rightAnswer) { selectedElement.classList.add('selected-green');
> 
> } else { selectedElement.classList.add('selected-red'); rightAnswer.classList.add('border');
> 
> } };

Screenshot 2022-10-12 at 19 59 11

As you can notice in the picture, I selected the correct answer in the first question, and the classList was added "selected-red" (in the Else option). In the second question, I selected the wrong answer , the classList was added "selected-red" (in the Else option)

**Review by arkaitzgarro — APPROVED** [review]

**Pull request merged**